src/helloworld.d.ts

```diff
@@ -5,12 +7,14 @@ export namespace helloworld {
 		setAge(n: number): void;
 		getAge(): number;
 		serializeBinary(): ArrayBuffer;
+		cloneMessage(): HelloRequest;
```

Comment:
added a missing function, this is necessary when wrapping in immutable structure. Refer `immut_helloworld.ts` file for more details